docs/community/events/csc23/index.md

```diff
@@ -0,0 +1,81 @@
+# CarbyneStackCon '23
+
+<p style="text-align: center; margin: 5em 0 5em 0;">
+  <img alt="CarbyneStackCon Logo" src="/images/events/csc/23/csc23-logo.png">
+</p>
+
+The _Carbyne Stack Conference_ (or _CarbyneStackCon_) is the annual meetup of
+the Carbyne Stack Open Source Community. It is designed to foster collaboration,
+discussion and knowledge sharing within the Carbyne Stack community.
+
+## Registration
+
+CarbyneStackCon '23 (CSC23) is an open event sponsored by Bosch Research taking
+place on November 30th. Everyone interested is welcome to attend! However,
+seats for in-person participation are limited and registration is required
+both for in-person and virtual attendance in order to facilitate our planning.
```

Comment:
is `both` swapped? -> `... is required for both in-person and...`

Reply:
Fixed